uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkPattern.php

```diff
@@ -73,6 +73,31 @@ private static function getMaxWeight() {
     return 0;
   }
 
+  /**
+   * Returns an array containing all the fields values for the
+   * WorkPattern with the given ID.
+   *
+   * This method is mainly used by the WorkPattern form, so it
+   * can get the data to fill its fields.
+   *
+   * An empty array is returned if it is not possible to load
+   * the data.
+   *
+   * @param int $id The id of the WorkPattern to retrieve the values
+   *
+   * @return array An array containing the values
+   */
+  public static function getDefaultValues($id) {
```

Comment:
I am not sure about the name of this method ... it used to get the values for any work pattern given its ID , so maybe getValues, getFieldValues or anything without the Default word in the name may be more appropriate